src/cljs/rems/application.cljs

```diff
@@ -760,26 +759,26 @@
                                        :name "Testers"
                                        :address "Testikatu 1, 00100 Helsinki"}
                           :application {:application/id 42
-                                        :application/applicant "developer"}}])
+                                        :application/applicant {:userid "developer"}}}])
    (example "member-info"
             [member-info {:element-id "info3"
                           :attributes {:userid "alice"}
                           :application {:application/id 42
-                                        :application/applicant "developer"}
+                                        :application/applicant {:userid "developer"}}
                           :group? true
                           :can-remove? true}])
    (example "member-info"
             [member-info {:element-id "info4"
                           :attributes {:name "John Smith"
                                        :email "john.smith@invited.com"}
                           :application {:application/id 42
-                                        :application/applicant "developer"}
+                                        :application/applicant {:userid "developer"}}
                           :group? true}])
 
    (component-info applicants-info)
    (example "applicants-info"
             [applicants-info {:application/id 42
-                              :application/applicant "developer"
+                              :application/applicant {:userid "developer"}
                               :application/applicant-attributes {:userid "developer"
```

Comment:
`:application/applicant-attributes` is still here

Reply:
done